src/util.js

```diff
@@ -32,7 +32,20 @@ function loadYamlFile(filePath) {
 }
 
 
+function loadFile(filePath) {
+  let fileData;
+  if (fs.existsSync(`${filePath}.js`)) {
+    fileData = require(`${filePath}.js`) // eslint-disable-line import/no-dynamic-require, global-require
+  } else if (fs.existsSync(`${filePath}.yaml`)) {
+    fileData = loadYamlFile(`${filePath}.yaml`)
+  } else if (fs.existsSync(filePath)) {
```

Comment:
This logic implies that files are loaded by first searching for .js and then .yaml (what about .yml?) and then just loading the file as is - which is assumed to be YAML. This search logic should be documented (I'm not sure if it is already, just saying it is something devs would need to know to figure out why their files fail to load)